content/wasm-languages/assemblyscript.md

```diff
@@ -101,144 +101,190 @@ directory '/Users/technosophos/Code/AssemblyScript/hello':
   ./package.json
   Package info containing the necessary commands to compile to WebAssembly.
 
-  ./index.js
-  Main file loading the WebAssembly module and exporting its exports.
-
   ./tests/index.js
-  Example test to check that your module is indeed working.
+  Stater test to check that the module is functioning.
+
+  ./index.html
+  Starter HTML file that loads the module in a browser.
 
 The command will try to update existing files to match the correct settings
-for this instance of the compiler in '/Users/technosophos/Code/AssemblyScript/hello/node_modules/assemblyscript'.
+for this instance of the compiler in '/Users/Fermyon/typescript-example/node_modules/assemblyscript'.
```

Comment:
Ditto the previous comment.  Can we make the path more system-agnostic?

Reply:
Done, thanks.